run_coder.py

```diff
@@ -0,0 +1,45 @@
+import torch
+import typer
+from pathlib import Path
+
+import settings
+import model
+from inference import INF
+
+
+def main(
+    path: Path = typer.Argument(
+        ...,
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+        readable=True,
+        resolve_path=True,
+        help="Path to file with fens",
+    ),
+    modelPath: Path = typer.Option(

```

Comment:
the name of this argument (modelPath) is inconsistent with the help text (path to config file)

Reply:
I forgot to change the description